lib/entice/web/client.ex

```diff
@@ -85,11 +87,35 @@ defmodule Entice.Web.Client do
     {:ok, friends}
   end
 
+  @doc "Returns a friend's online status and character name from his account id."
+  def get_status(account_id) do
+    case Entice.Web.Repo.get(Entice.Web.Account, account_id) do
+      nil -> {:error, :no_matching_account}
+      acc ->
+        acc = Entice.Web.Repo.preload(acc, :characters)
+        case Client.Server.get_client(acc.email) do
+          nil ->
+            if length(acc.characters) == 0 do
```

Comment:
this if would be better placed in a guard, or maybe we can avoid it with a match